microprofile.jdt/com.redhat.microprofile.jdt.quarkus/src/main/java/com/redhat/microprofile/jdt/internal/quarkus/providers/QuarkusConfigRootProvider.java

```diff
@@ -359,11 +359,11 @@ private void addItemMetadata(String extensionName, IField field, String fieldTyp
 		// Default value for primitive type
 		if ("boolean".equals(fieldTypeName)) {

```

Comment:
Add in JDTTYpeUtils a static method `isBoolean(String valueClass)`  (like isNumber)

Reply:
How's this?
```
public static boolean isBoolean(String valueClass) {
	return valueClass.equals("boolean") || valueClass.equals("java.lang.Boolean");
}
```

Do we want both `boolean` and `java.lang.Boolean`?